[TOREVIEW] Forestier_Herzig

QuentinForestier opened the pull request:
Work in progress.
We are coming back soon with update.

QuentinForestier committed:
Implémentation de utils

Tous les tests passent

QuentinForestier committed:
Merge pull request #1 from QuentinForestier/fb_utils_implementation

Implémentation de utils

QuentinForestier committed:
Implementation UpperCaseFilterWriter

Tous les tests passent

QuentinForestier committed:
Merge pull request #2 from QuentinForestier/fb_uppercasefilterwrite_implementation

Implementation UpperCaseFilterWriter

MelvynHerzig committed:
Validation du test 2 et 4 de ApplicationTest

Les test 1 et 3 ne fonctionnnent pas encore

MelvynHerzig committed:
Merge pull request #3 from QuentinForestier/fb_application_implementation

Le code application a été implémenté

MelvynHerzig committed:
Implémentation FileTransformer et NoOpFileTransformer

Les classes ont été implémentées et les tastes passent.

MelvynHerzig committed:
Merge pull request #4 from QuentinForestier/fb_FileTransformer_implementation

Implémentation FileTransformer et NoOpFileTransformer

MelvynHerzig committed:
Implémentation du FileNumberingFilterWriter

Les tests passent

MelvynHerzig committed:
Merge pull request #5 from QuentinForestier/fb_fileNumberingFilterWriter_implementation

Implémentation du FileNumberingFilterWriter

MelvynHerzig committed:
completeFileTransformer implémenté

Les tests passent

MelvynHerzig committed:
Merge pull request #6 from QuentinForestier/fb_CompleteFileTransforme_implementation

completeFileTransformer implémenté

MelvynHerzig committed:
Compiler version et exclude info class

 Il manquait la version au plugin compiler, nous l'avons ajouté sinon maven ne fonctionnait pas, la balise exclude module-info class a été ajouté, le jar ne buildait pas pas sinon

MelvynHerzig committed:
Merge pull request #7 from QuentinForestier/fb_ajustement_pom

Compiler version et exclude info class

MelvynHerzig committed:
Les tests ne doivent pas utiliser la version int

MelvynHerzig committed:
Refactor de FilterNumberWriter

Les méthodes avec String et char[] n'utilisent plus la méthode write(int)

MelvynHerzig committed:
Merge pull request #8 from QuentinForestier/fb_Amelioration_FilterNumberWriter

Fb amelioration filter number writer

MelvynHerzig committed:
DFSFileExplorer tests adjusted

The files list is now sorted

MelvynHerzig committed:
Merge pull request #9 from QuentinForestier/fb_Adjusting_dfsExplorer_Tests

DFSFileExplorer tests adjusted

MelvynHerzig commented:
Greetings,

We are done implementing the java-io-lab.

Thanks for everything, have a nice day!

MelvynHerzig committed:
Ajout des IO avec des buffers

QuentinForestier committed:
getNextLine remasterized

Now it is possible to have multiple delimiter style AND the array of splitter is sorted as wanted by the code

QuentinForestier committed:
Merge pull request #10 from QuentinForestier/fb_Improving_IO_With_Bffer

Fb improving io with bffer

MelvynHerzig commented:
Greetings,

We are done implementing the java-io-lab

We encountered an unexpected problem but we solved it. We are cracks

Thanks for everything, have a nice day.
